docs/hub/how-to-inference.md

```diff
@@ -0,0 +1,44 @@
+# How to integrate the Inference API in your library
```

Comment:
I'm not sure we should have the Inference API client in the Hub documentation at the moment since we're not pushing much for it.

@julien-c @LysandreJik WDYT?

Reply:
I think it's good to advertise it here! Even if we're not pushing for it, it would be interesting to have some users in order to have a potential v2 of it when (if) it is moved. We can't have feedback if users don't use it, and users won't use it if it's not documented.

Reply:
This section reads quite nicely. I'm growing fond of it :)

Reply:
this is a bit redundant with https://huggingface.co/docs/hub/inference#is-there-programmatic-access-to-the-inference-api, this one should at least link to here

Reply:
We should also link from the Spaces doc to mention this, BTW

Reply:
Finally I suggest a name change for this section in https://github.com/huggingface/huggingface_hub/pull/385/files#r723134175

Reply:
Good point on the naming, how does this sound: `How to programmatically access the Inference API`?

To reduce redundancy, I'd suggest removing the shorter `Is there programmatic access to the Inference API?` section in favor of the slightly more detailed one. I think this would actually make this topic easier for users to discover, especially after we rename the section :)